api/src/main/java/org/pmiops/workbench/db/model/User.java

```diff
@@ -96,14 +97,23 @@ public void setContactEmail(String contactEmail) {
   }
 
   @Column(name = "data_access_level")
-  public DataAccessLevel getDataAccessLevel() {
+  public Short getDataAccessLevel() {
```

Comment:
In other places you've got Short on the ends of getters and setters... want to do that in this file too?

Once you've got the Short suffix everywhere, you arguably could leave the "Enum" off the getters / setters that return enums and have them use the original method names.  (Assuming that works with JPA.) Though being explicit about enums is fine, too.

Reply:
That was accidental, thanks. I reverted that approach because the naming caused problems in combination with the Dao FETCH JQL stuff. I didn't look to deeply into whether there were workarounds, but what I have now seems like the safest route.